src/sentry/search/events/filter.py

```diff
@@ -1205,3 +1208,22 @@ def _issue_id_filter_converter(self, search_filter: SearchFilter) -> Optional[Wh
         # Skip isNull check on group_id value as we want to
         # allow snuba's prewhere optimizer to find this condition.
         return Condition(lhs, Op(search_filter.operator), rhs)
+
+    def _error_handled_unhandled_filter_converter(
```

Comment:
Am I missing something here? Why is the existing [implementation](https://github.com/getsentry/sentry/blob/master/src/sentry/search/events/filter.py#L263-L297) so complicated? According to [snuba](https://github.com/getsentry/snuba/blob/master/snuba/query/processors/handled_functions.py#L23), they should be mutually exclusive... Does the original method query faster or something?